pyproject.toml

```diff
@@ -7,18 +7,19 @@ requires-python = ">=3.12"
 dependencies = [
     "beautifulsoup4==4.*",
     "cdsapi==0.7.5",
-    "lxml==4.9.*",
-    "py7zr==0.20.6",
-    "requests>=2",
+    "freezegun==0.3.4",
```

Comment:
separate testing deps into its own section, see for example https://github.com/globaldothealth/adtl/blob/8d282f5397ccbafcb2dd54a28aaf1caf0a85d724/pyproject.toml#L58